Lib/email/headerregistry.py

```diff
@@ -542,6 +554,7 @@ def cte(self):
     'content-type':                 ContentTypeHeader,
     'content-disposition':          ContentDispositionHeader,
     'content-transfer-encoding':    ContentTransferEncodingHeader,
+    'message-id':                   MessageIDHeader,
```

Comment:
Slightly offtopic to PR but during review I find `_default_header_map` is documented at https://docs.python.org/3/library/email.headerregistry.html#email.headerregistry.HeaderRegistry but is out of sync with the current default map. Should this be updated? If so it can be done as separate PR.

Reply:
Yes, it should be updated.  And you might as well do it in this PR, since that doc should be updated anyway to reflect this new feature :)

Reply:
Added `message-id` and all the missing headers too. Thanks @tirkarthi for pointing it out.